src/core/unit.c

```diff
@@ -2308,6 +2308,7 @@ static int unit_log_resources(Unit *u) {
 void unit_notify(Unit *u, UnitActiveState os, UnitActiveState ns, bool reload_success) {
         Manager *m;
         bool unexpected;
+        bool special;
```

Comment:
Hmm, this name needs to change to reflect the meaning (`release_console_but_remain` or so).